components/p3a/brave_p3a_service.cc

```diff
@@ -89,6 +89,7 @@ constexpr const char* kCollectedHistograms[] = {
     "Brave.Today.WeeklyMaxCardViewsCount",
     "Brave.Today.WeeklyMaxCardVisitsCount",
     "Brave.Sync.Status",
+    "Brave.Sync.TypesEverReenabled",
```

Comment:
have you got a privacy review for the new metric?

Reply:
No, I didn't know it is required. Thanks, will request that.

Reply:
Request for a privacy review is submitted.